x-pack/platform/plugins/private/file_upload/public/importer/importer.ts

```diff
@@ -250,6 +290,20 @@ export abstract class Importer implements IImporter {
       body,
     });
   }
+
+  public async deletePipelines(pipelineIds: string[]) {
+    // remove_pipelines
+    const body = JSON.stringify({
+      pipelineIds,
+    });
+
+    return await getHttp().fetch<IngestDeletePipelineResponse[]>({
+      path: `/internal/file_upload/remove_pipelines`,
+      method: 'POST',
```

Comment:
I reckon we should use DELETE here

Reply:
Updated in https://github.com/elastic/kibana/pull/206864/commits/8bf59f8a149da53e5f73aa55cf472bf9df13c11b